### DISCUSSION TITLE
Kann ich das AIOTE Device ausschalten, wenn ich über längere Zeit keine Veränderung des Zählers erwarte?

### DISCUSSION THREAD

**Discussion by AdmiralRaccoon** [jomjol/AI-on-the-edge-device#2620] [General]
Hallo,
sinnfreier Titel, aber ich wusste nicht wie ichs am besten beschreibe:

Ich habe im Grunde seit April keine Veränderung auf der Gasuhr, da die Wärmepumpe momentan alles macht, denke mal ab November wird auch wieder Gas hinzukommen.

Sollte ich während dieser Zeitspanne, der Kontinuität halber, den AOTE einfach laufen lassen, und 1x am Tag eine Ablesung machen, oder einfach komplett vom Strom nehmen? Da weiss ich dann z.b. nicht ob das Energy Dashboard voom Home Assistant dann durcheinander kommt.

Danke & Grüsse
Philipp

**Comment by caco3** [comment]
Du kannst ihn einfach ausschalten. Das Energy Dashboard nimmt einfach die MQTT Daten, die es kriegt, wenn nichts kommt, wird es einfach 0 anzeigen.
Wichtig ist, dass du dein AIOTE Device wieder einschaltest, bevor die Heizung wieder viel arbeitet, da Du sonst einen grossen Sprung sehen wirst.

**Reply by AdmiralRaccoon** [reply]
Super, danke für die Info!
